todoman/cli.py

```diff
@@ -336,6 +337,9 @@ def list(
 
     sort = sort.split(',') if sort else None
 
+    if priority not in ['medium', 'low', None, 'high']:
+        raise "Invalid priority"
```

Comment:
Additionally, I'd also prefer if this validation function returned the numeric priority based on the input string as well, so converting string -> number is done strictly in the `cli` code..